src/server/saved_objects/service/lib/repository.js

```diff
@@ -224,6 +224,35 @@ export class SavedObjectsRepository {
     );
   }
 
+  async deleteByQuery(options = {}) {
```

Comment:
nit: Can we add some JSDoc comments for this?

Reply:
not a nit, will do!